src/Kestrel.Core/Adapter/Internal/AdaptedPipeline.cs

```diff
@@ -108,7 +102,7 @@ public class AdaptedPipeline : IDuplexPipe
             }
             catch (Exception ex)
             {
-                error = ex;
+                Log.LogError(0, ex, $"{nameof(AdaptedPipeline)}.{nameof(WriteOutputAsync)}");
```

Comment:
This should only happen now if writing to the stream fails, yeh

Reply:
Correct. This would be really bizarre given that Stream.WriteAsync should never fail on the RawStream given that no CancellationToken is passed here, but not strictly impossible.